tests/deploy-values.yaml

```diff
@@ -1,3 +1,5 @@
+global:
+  defaultVectorStoreType: pgvector
```

Comment:
chromadb is still enabled for now, can we update it to false if we'll use pgvector by default now?